create.go

```diff
@@ -289,3 +302,47 @@ func createPIDFile(pidFilePath string, pid int) error {
 
 	return nil
 }
+
+// copyParentCPUSet copies the cpuset.cpus and cpuset.mems from the parent
+// directory to the current directory if the file's contents are 0
+func copyParentCPUSet(current, parent string) error {
+	currentCpus, currentMems, err := getCPUSet(current)
+	if err != nil {
+		return err
+	}
+
+	parentCpus, parentMems, err := getCPUSet(parent)
+	if err != nil {
+		return err
+	}
+
+	if len(parentCpus) < 0 || len(parentMems) < 0 {
```

Comment:
This looks odd - shouldn't this be checking the value of these variables rather than their length?

Reply:
```
#cat /sys/fs/cgroup/cpu/cpuset.cpus 
0-23
```
value of cpuset.cpus can be 0-1 / 0 /1,2,3   , so I think use `len(parentCpus) < 0` maybe properly